hack/addons/kustomize/cilium/kustomization.yaml.tmpl

```diff
@@ -15,10 +15,13 @@ helmCharts:
   repo: https://helm.cilium.io/
   releaseName: cilium
   version: ${CILIUM_VERSION}
-  valuesFile: helm-values.yaml
+  valuesFile: ../../../../charts/cluster-api-runtime-extensions-nutanix/addons/cni/cilium/values-template.yaml
   includeCRDs: true
   skipTests: true
   namespace: kube-system
   kubeVersion: ${E2E_KUBERNETES_VERSION}
+  valuesInline:
```

Comment:
Please add a comment why this is needed in CRS.

Reply:
Maybe we should also put this next to `valuesFile`?

Reply:
Added a comment and moved values file and values inline next to each other.